HTTPServerProject/WriteStreams.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+using System.Collections.Generic;
+
+namespace HTTPServerProject.WriteStreams
+{
+    public interface IWriteStreams
+    {
+        void WriteLine(string str = default!);
+        void Flush();
+        void Close();
+    }
+
+    public class MyStreamWriter : IWriteStreams
```

Comment:
Let's get a better name than "My"StreamWriter on this.

Reply:
Updated to `WriteStreams` and test is now `TestWriteStreams`. I also updated `MyStreamReader` to `ReadStreams`